tex/chapters/literature.tex

```diff
@@ -1,11 +1,274 @@
+This is then followed by a few examples.
+The examples presented are of low quality and the only probe strategy (see section/definition etc) used is TitForTat.
+%TODO
+
+Ashlock then extends his fingerprinting work further in ~\cite{Ashlock2008}.
+Far more examples are presented, and many different fingerprint functions are listed.
+A large number of the analytical fingerprint functions use a probe that is not TitForTat.
+Fingerprinting is then used to to assess how three evolutionary algorithms produce different populations.
+The evolutionary methods involved are finite-state machines, lookup tables and feedforward neural nets.
+
+
+\begin{definition}\label{def:joss-ann}
+If $A$ is a strategy for playing the iterated prisoner's dilemma, then the \textbf{Joss-Anne of A}, $\JA(A, x, y)$ is a transformation of that strategy.
+Instead of the original behaviour, it makes move $C$ with probablility $x$, move $D$ with probability $y$, and otherwise uses the response appropriate to strategy $A$ (if $x+y < 1$).
+\end{definition}
+
+The notation $\JA$ comes from the initials of the names Joss and Anne.
+Joss was a strategy submitted to one of Axelrod’s original tournaments and it would occasionally defect without provocation in the hopes of a slight improvement in score.
+Anne is the first name of A. Stanley who suggested the addition of random cooperation (refs from ashlock paper) instead of random defection ~\cite{Ashlock2008}.
+When $x + y = 1$, the original strategy is not used, and the resulting behavior is a random strategy with probabilities $(x, y)$.
+In more general terms, a $\JA$ strategy is an alteration of a strategy $A$ that causes the strategy to be played with random noise inserted into the responses.
+
+\begin{definition}\label{def:fingerprint}
+A \textbf{Fingerprint} $F_A(S, x, y)$ with $0 \leq x, y \leq 1$, $x+y \leq 1$ for strategy $S$ and probe $A$, is the function that returns the expected score of strategy $S$ against $\JA(A, x, y)$ for each possible $(x, y)$.
+\end{definition}
+
+The fingerprint function can often be found quite easily (see Chapter \ref{cha:theory} for an example construction) however it is the plot of this function that is of interest.
+This is because an approximation of the plot can be found in cases where the analytical function would be too difficult or time consuming to compute.
+
+\begin{definition}\label{def:double-fingerprint}
+The \textbf{Double Fingerprint} $F_{AB}(S, x, y)$ with $0 \leq x, y \leq 1$ returns the expected score of strategy $S$ against $\JA(A, x, y)$ if $x+y \leq 1$, and $\JA(B, 1-y, 1-x)$ if $x+y \geq 1$.
+\end{definition}
+
+The double fingerprint allows us to extend the idea of a fingerprint over the full unit square.
+A unit square is preferable because it is more easily manipulated by a computer, is more easily viewed by humans, and uses paper more efficiently. %TODO, do we like this line
+
+\begin{definition}\label{def:dual}
```

Comment:
Make it clear that these definitions are not yours (so reference them, perhaps in that introductory sentence which is currently not here).